src/Reference/HasOne_SQL.php

```diff
@@ -45,8 +48,12 @@ public function addField($field, $their_field = null) : Field_SQL_Expression
             $their_field = $field;
         }
 
+        // if caption is not defined in $defaults -> get it directly from the linked model field $their_field
+        $defaults['caption'] = $defaults['caption'] ?? $this->owner->refModel($this->link)->getField($their_field)->getCaption();
```

Comment:
I plan to implement this differently by using new `Field->getSeed()` method to pass all field properties (including caption) to new field. Anyway this is good for now, but will conflict with my PR later.

Reply:
OK, IMHO what is needed is to not redefining "caption" for reference imported fields, when you will break it we will go in this point and change the code, but final implementation will not have to change nothing.
What you think if i add a Test for this case, to see when this BC happen?

Reply:
Add Test. Not bad idea to have a test even for small things.